src/metabase/task/backfill_query_fields.clj

```diff
@@ -10,19 +10,21 @@
 
 (set! *warn-on-reflection* true)
 
-(defn- backfill-query-fields! []
-  (let [cards (t2/reducible-select :model/Card :id [:in {:from      [[:report_card :c]]
```

Comment:
The semantics of this query are the same, except it no longer uses a subselect. This means it is both more efficient, and (more importantly) easier to extend as the sweeper gets smarter.